package/MDAnalysis/core/universe.py

```diff
@@ -238,6 +238,17 @@ class Universe(object):
     vdwradii: dict, ``None``, default ``None``
         For use with *guess_bonds*. Supply a dict giving a vdwradii for each
         atom type which are used in guessing bonds.
+    context: str or Guesser, default ``default``
+        Type of the Guesser to be used in guessing TopologyAttrs
+    to_guess: list[str], (optional), default ``['types', 'masses']``
+              (in future versions types and masses will be removed)
+        TopologyAttrs to be guessed. These TopologyAttrs will be wholly guessed
+        if they don't exist in the universe. If they already exist in the Universe,
+        only empty or missing values will be guessed.
```

Comment:
I would add an additional block with an explicit deprecation warning:
```reST

.. warning::

   In MDAnalysis 2.x, types and masses are being automatically guessed if 
   they are missing (``to_guess=('types, 'masses')``). However, starting
   with release 3.0 **no guessing will be done by default** and it will be
   up to the user to request guessing using ``to_guess`` and ``force_guess``.
```